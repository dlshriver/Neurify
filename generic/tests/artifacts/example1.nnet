// FORMAT::
// numLayers,inputSize,outputSize,maxLayerSize
// layerSize_0,layerSize_1,...,layerSize_{numLayers}
// layerType_1,layerType_2,...,layerType_{numLayers} // (0: fully connected, 1: convolutional)
// foreach convolutional layer: numOutChannels,numInChannels,kernelSize,stride,padding
// foreach layer: weights,bias
3,2,2,2
2,2,2,2
0,0,0
1.0,1.0,
1.0,-1.0,
0.0,
0.0,
1.0,1.0,
1.0,-1.0,
0.0,
0.0,
1.0,1.0,
0.0,1.0,
1.0,
0.0