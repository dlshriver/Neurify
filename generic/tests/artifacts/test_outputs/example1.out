-0.3,-0.01,
0.2,1.0,
